uk.co.compucorp.civicrm.tasksassignments/tasksassignments.php

```diff
@@ -463,3 +468,22 @@ function _tasksAssignments_remove_non_civihr_tabs_from_workflow (phpQueryObject
     $doc->find('#acttab-' . $tab)->remove();
   }
 }
+
+/**
+ * Removes the Workflow configuration's actions dropdown and replaces it with a
+ * button that only allows the "Add timeline" action.
+ *
+ * @param phpQueryObject $doc
+ */
+function _tasksAssignments_allow_only_add_timeline_action (phpQueryObject $doc) {
+  $actionDropdown = $doc->find('select[ng-model="newActivitySetWorkflow"]');
```

Comment:
Was there any reason why you did not go the approach of probably removing the unwanted option values in the select dropdown rather than creating a new button element.
I'm of the opinion that the latter would require minimal changes should anything change in the core code and also ensure that the html structure from core is preserved.
What do you think?

Reply:
@tunbola having a single "Add timeline" button was part of the requirement ([wiki link](https://compucorp.atlassian.net/wiki/spaces/PCHR/pages/134906233/Tasks+and+Workflows+-+Administrator+UX+improvements)), but even then, I think removing the dropdown is the way to go. 

If we only remove the options that we don't need it will leave only a single option in the dropdown (even if that might change in the future) so to allow users to add timelines we'd either need to add some sort of "run selected action" button next to the dropdown or add an empty "Please select an action" button.

Since we'll only have one available action for now it makes sense to turn it into a button.